shared-ts/jsonTypes.ts

```diff
@@ -106,7 +106,7 @@ export enum MiscellaneousReason {
     undefinedProblem,
     problemsAtBorderPost,
     problemsAtCustomsPost,
-    problemsOnLocalRoad,
+    problemsOnLocalRoad
```

Comment:
Prettier will want this trailing comma, it's worth installing the prettier and eslint plugins for vscode if you don't have them already

Reply:
done